Formula/ipmitool.rb

```diff
@@ -27,20 +25,6 @@ class Ipmitool < Formula
     depends_on "readline"
   end
 
-  # fix enterprise-number URL due to IANA URL scheme change
-  # remove in next release
-  patch do
-    url "https://github.com/ipmitool/ipmitool/commit/1edb0e27e44196d1ebe449aba0b9be22d376bcb6.patch?full_index=1"
-    sha256 "c7df82eeb6abf76439ca9012afdcef2e9e5ab5b44d4a80c58c7c5f2d8337bc83"
-  end
-
-  # Patch to fix build on ARM
-  # https://github.com/ipmitool/ipmitool/issues/332
-  patch do
-    url "https://github.com/ipmitool/ipmitool/commit/a45da6b4dde21a19e85fd87abbffe31ce9a8cbe6.patch?full_index=1"
-    sha256 "98787263c33fe11141a6b576d52f73127b223394c3d2c7b1640d4adc075f14d5"
-  end
-
```

Comment:
which patches removal, maybe bump the `revision` as well?

Reply:
was work in progress, is now fixed

Reply:
(and pushed when the tain wifi is working)